arm9/src/render_text.c

```diff
@@ -0,0 +1,501 @@
+#include "global.h"
+#include "render_text.h"
+#include "game_init.h"
+#include "string_util.h"
+#include "text.h"
+#include "unk_0201B8B8.h"
+
+u16 unk00;
+
+TextFlags gTextFlags;
+
+const u8 UNK_020ECB50[] = { 0, 1, 2, 1 };
+
+extern const char *FUN_02002D94(u8, u16);
+extern void PlaySE(u16);
+
+THUMB_FUNC u32 RenderText(struct TextPrinter *printer)
+{
+
+    struct TextPrinterSubStruct *subStruct =
+        (struct TextPrinterSubStruct *)(&printer->subStructFields);
+    u16 currentChar;
+
+    switch (printer->state)
+    {
+    case 0:
```

Comment:
indentation here

Reply:
How do you want the indentation to be? It's set this way by clang-format

Reply:
cases etc should be one in from the { imo, it's fine as is though I guess

Reply:
It's specifically set to false here: https://github.com/pret/pokediamond/blob/54b6645c02958cff375393a5261d5376dd169b94/.clang-format#L11

But I can change that if you want

Reply:
might be a good idea

Reply:
done